python-lists-and-tuples/solution.py

```diff
@@ -0,0 +1,9 @@
+L = 'abcdefghijklmnopqrstuvwxyz'
+def lpass(alphabet):
+  new_list = [ ]
+  b_list = list(range(1,len(alphabet)+1))
+  for i in alphabet:
+    new_list = new_list + [(b_list[alphabet.index(i)],i)]
```

Comment:
This seems a bit overly complicated for what needs to be achieved. I would recommend using enumerate() in the for loop to obtain a tuple with the index and letter of each letter in the alphabet. Hint: use the "start" parameter of enumerate().